src/admin/adminUI/listModal/components/AdminStudentList.js

```diff
@@ -10,6 +13,17 @@ import { ButtonText } from "../../menuButton/ButtonText";
 const AdminStudentList = () => {
     const [shouldDisplay, setShouldDisplay] = useState(false);
     const [addFormType, setAddFormType] = useState("");
+    const [students, setStudents] = useState([]);
+    const [searchName,setSearchName] = useState("");
+
+
+	useEffect(()=> {
```

Comment:
The indentation is wrong. Please make useEffect() align with const. You can use vscode plugin like Prettier to adjust the format of your coding.